### ISSUE TITLE
Consider aligning icon colours with popular logos

### ISSUE THREAD

**Issue by humphd** [mozilla/thimble.mozilla.org#2009]
I was playing with some files from @Pomax in Thimble, which were mostly JS:

![screen shot 2017-04-20 at 12 27 27 pm](https://cloud.githubusercontent.com/assets/427398/25241666/d7232f26-25c4-11e7-8cf9-4c2fd3ad6c86.png)

I thought it ended up looking really orange.  In my mind "orange" is [HTML5](https://www.w3.org/html/logo/):

![](https://www.w3.org/html/logo/img/html5-display.png)

And JS is yellow:

![](https://i.stack.imgur.com/Mmww2.png)

We already use blue for CSS, which seems pretty common now too.

@flukeout What do you think of aligning these with the popular icon colours more closely?

**Comment by flukeout** [comment]
@humphd Yeah, I chose those colors to match the "badge icons" for CSS/HTML/JS, the trouble with the JS yellow is that it was too light for the light theme and looked out of place next to the others once, that's why I darkened it.

I think I muted the orange for the HTML one because it seemed kind of warning-y and very bright relative to the others.

We _could_ create a separate icon for it that is a bit darker for the light theme but not sure it's worth the effort.

Anyway, I can revisit at a later point.
